rosdep/python.yaml

```diff
@@ -4481,6 +4481,13 @@ python-rx-pip:
   ubuntu:
     pip:
       packages: [rx]
+python-sbp-pip:
```

Comment:
do you want to make it available for python3?
Please use `python3-sbp-pip` in that case.
Remember to alpha-order again

Reply:
Just added an alias for python3 to refer to the python2 version